Game_Doc.md

```diff
@@ -0,0 +1,25 @@
+# Concentration Game
+## Description
+A simple concentration card game where the user flips cards to reveal it's icon and must find that cards duplicate by flipping another card.
+
+## Components
+* [Board Code](./components/board.tsx) / [Doc](#board)
+* [Card Code](./components/card.tsx) / [Doc](#card)
+
+## Board
```

Comment:
It's usually good to talk to someone who is trying to use your project/components. An example of using the Board component is good:

```tsx
<Board size={[4, 3]} />
```

Letting them know the expected shape of the `size` property is very helpful. At a glance, someone (familiar with Typescript) can look at this Interface* and quickly reason it out:

```ts
interface BoardProps {
  size: [width: number, height: number]
}
```

\* I haven't updated the Board component's Interface the this but it would be a good idea.